Laba4/kNN.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
```

Comment:
Этот файл уже был. Зачем он здесь повторно? Удалить

Reply:
```suggestion
import numpy as np
```
Этот файл используется в 10 стр. 
`sub_X = sub_X - np.mean(sub_X)`